astro/src/content/quickstarts/quickstart-dotnet-web.mdx

```diff
@@ -0,0 +1,325 @@
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/Pages/Logout.cshtml"}
+  lang="razor" />
+
+### Application Configuration
+You will need to do a little plumbing to ensure things like having the correct settings and configuring the cookies properly to work over http for the sample application.
+
+#### App Settings
+Update the `appsettings.json` file. You need to add the entire `complete-application` object so that the code above can be configured correctly. `Authority` is just the location of the user identity server, in this case, FusionAuth.  `Urls` ensures the application runs on the specified port, 5000.
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/appsettings.json"}
+  lang="json" />
+
+#### Program Settings
+Update the `Program.cs` file. Since you are using unsecured cookies, you need to allow this in the configuration.  Safari and Chrome require different settings and that is reflected in the code below.
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/Program.cs"}
+  lang="c#" />
+
+#### Startup Settings
+Add the `Startup.cs` file in the root directory of the application. This sets the correct port and manages unsecured cookies for the browsers.  The `CheckSameSite` and `DisallowsSameSiteNone` functions help manage the cookie settings for the different browsers.
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/Startup.cs"}
+  lang="c#" />
+
+<Aside type="note">
+There are a few settings applied to simplify the sample application. In a production application, do not allow cookies over `http` or keep the `ClientSecret` in a .json file for production applications.  Get the `ClientSecret` from an environmental variable that has been set on a production machine or some other secure way.
+</Aside>
+
+#### Navigation
+This application will display different menu items for authenticated users and non authenticated users. To add this type of navigation update `/Pages/Shared/_Layout.cshtml` with the following:
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/Pages/Shared/_Layout.cshtml"}
+  lang="razor" />
+
+#### Homepage
+Update the default page.
+
+Copy the following code into `Index.cshtml.cs`:
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/Pages/Index.cshtml.cs"}
+  lang="c#" />
+
+Copy the following code into `Index.cshtml` and notice the logic to show a different view to users that have not logged in.
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/Pages/Index.cshtml"}
+  lang="razor" />
+
+#### Styling
+To give the pages the look and feel of an application, apply the following styling and images.
+
+Add the `changebank.css` file in the `wwwroot\css" directory of the application.
+
+<RemoteCode url={frontmatter.coderoot + "/complete-application/wwwroot/css/changebank.css"}
+  lang="css" />
+
+Add the Changebank logo to the application.
```

Comment:
Is this missing a command?

Reply:
the page is pulling from https://fusionauth.io/assets/img/samplethemes/changebank/changebank.svg so don't think we need this line at all.